### PR TITLE
chore: Move name converter and improve imports

### PR DESCRIPTION
**Pull request by marikaner**
As in the title. This should be fairly simple to review.

### CODE REVIEW

**Review by FrankEssenberger — APPROVED** [review]
LGTM

File: packages/util/src/string-formatter.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  return voca.camelCase(str);
+}
+
+/**
+ * Convert a string to a human readable format, e.g. it transforms `to_BusinessPartner` to `To Business Partner`.
+ *
+ * @param str - The string to be transformed.
+ * @returns The transformed string.
+ */
+export function toTitleFormat(str: string): string {
```
Comment (marikaner) [inline packages/util/src/string-formatter.ts:29]: This function has a slightly different name than the others. The reason is, that we might want to add a `toTitleCase` function that transforms `content-type` to `Content-Type` as well.

**Comment by jjtang1985** [comment]
👍 we should do the same for all unit tests.

File: packages/util/src/string-formatter.ts (hunk 1 above)
Comment (FrankEssenberger) [reply packages/util/src/string-formatter.ts:29]: fine for me. `Format` is a good word since it is not changing the case.

**Pull request merged**